src/FCharacter.cpp

```diff
@@ -22,6 +22,12 @@ FCharacter::FCharacter(FVector position, FVector velocity, std::vector<FWeapon>
 	m_type = type;
 }
 
+bool FCharacter::isAlive()
+{
+	//TODO
+	return true;
```

Comment:
I AM THE BAWS. :)
Yeah, that sounds reasonable. Int variables for lives, maxHealth, currentHealth, and then each weapon/projectile will have a hurt/damage variable.